packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewerContainer.js

```diff
@@ -113,15 +116,19 @@ class VariableStarViewerContainer extends Component {
 
   onLoad (rawJSON) {
     const {
-      scatterPlot,
-      barCharts
+      data: {
+        scatterPlot,
+        barCharts
+      }
     } = rawJSON
+
     const { onReady, subject } = this.props
     const target = this.viewer.current
     const phasedJSON = this.calculatePhase(scatterPlot)
     const barJSON = this.calculateBarJSON(barCharts)
     const visibleSeries = this.setupSeriesVisibility(scatterPlot)
-    const imageLocation = subject.locations.find(location => location['image/png']) || {}
+    // think about a better way to do this
+    const imageLocation = subject.locations[2] || {}
```

Comment:
This does seem a bit odd to me. Are we still looking for the `'image/png'` here and can we always assume it will be at that position?

Reply:
Yes, it's brittle. I need to think about a better way to do this, but the best I can think of is relying on the order of the locations array for which image we want to display here. The issue is that they need a fallback image for talk to be first and the second image will be for actual classification.